packages/valory/connections/abci/tests/test_fuzz/test_fuzz.py

```diff
@@ -32,6 +34,11 @@
 )
 
 
+running_on_ci = os.getenv(CI)
+if running_on_ci:
+    settings.load_profile(CI)
```

Comment:
which is then loaded in each of the relevant files
https://github.com/valory-xyz/open-autonomy/blob/9624a1b64007148192a518239139bdb9c3f5134f/packages/valory/skills/abstract_round_abci/tests/test_behaviours_utils.py#L109